mods/README.md

```diff
@@ -0,0 +1,17 @@
+# Mods
+
+## Limitations
+The purpose and abbilities of Mods are restricted to modifying the `pokete_data` module.
```

Comment:
"Abbilities" seems to be a typo? Should be "abilities".

Reply:
Oh, yes, I just fixed it.
Thanks a lot